mozci/console/commands/push.py

```diff
@@ -172,23 +168,17 @@ def handle(self) -> None:
             )
 
         for push in pushes:
-            try:
-                classification, regressions = push.classify(
-                    intermittent_confidence_threshold=medium_conf,
-                    real_confidence_threshold=high_conf,
-                )
-                if retrigger_unknown:
-                    for _, tasks in regressions.unknown.items():
-                        retrigger(tasks=tasks, repeat_retrigger=1)
-                self.line(
-                    f"Push associated with the head revision {push.rev} on "
-                    f"the branch {self.branch} is classified as {classification.name}"
-                )
-            except Exception as e:
-                self.line(
-                    f"<error>Couldn't classify push {push.push_uuid}: {e}.</error>"
-                )
-                continue
```

Comment:
Here we were actually ignoring the error, maybe because if we fail classifying one out of say 100 pushes we don't want to totally stop. Perhaps we can just print the stacktrace here in addition to the error and still continue?

Reply:
We can also merge this as-is and figure out if this ever happens

Reply:
Yep that was the point of the logged error + `continue`, do you want me to rollback this change and just log the stacktrace in addition?